docs/sdk/api/typescriptSdk.md

```diff
@@ -25,6 +25,115 @@ npm install --save @commercetools/ml-sdk
 </script>
 ```
 
+## Getting started
+This tutorial will show you how to use the middlewares in this **[commercetools TypeScript SDK](https://github.com/commercetools/commercetools-sdk-typescript/)** to get a simple commercetools JavaScript app running.
+
+### Create a API client
+[Create API client](https://docs.commercetools.com/tutorials/getting-started#creating-an-api-client) from Merchant Center. If you do not have account [follow the steps to create a free trial account](https://docs.commercetools.com/tutorials/getting-started#first-steps). 
+In this guide we’ll be calling a method of commercetools API using TypeScript SDK to get the settings of a commercetools project. The commercetools API is the foundation of the commercetools Platform, and almost every commercetools client app uses it. Aside from getting project information, the commercetools API allows clients to call methods that can be used for everything from creating products to updating an order’s status. Before we can call any methods, we need to configure our new app to obtain an access token.
+
+### Set up your local project
+If you don’t already have a project, let’s create a new one. In an empty directory, you can initialize a new project using the following command:
```

Comment:
Can we say "Node.js project" here? I just want to be clear it cannot be confused with the commercetools project that should be existing at this step for sure.

Reply:
Yes